solutions/dedicated/README.md

```diff
@@ -0,0 +1,13 @@
+# IBM Cloudant Dedicated on IBM Cloud Dedicated
+
+This architecture creates an IBM Cloudant Dedicated instance in IBM Cloud Dedicated and supports provisioning the following resources:
```

Comment:
I think this reads better: `in an IBM Cloud Dedicated environment`

Reply:
Can we leave it as is because not everything is in the dedicated environment - like resource group

Reply:
I was suggesting: 

This architecture creates an IBM Cloudant Dedicated instance in an IBM Cloud Dedicated environment and supports provisioning the following resources